vllm/engine/arg_utils.py

```diff
@@ -720,8 +720,11 @@ def add_cli_args(parser: FlexibleArgumentParser) -> FlexibleArgumentParser:
             "--mm-processor-kwargs",
             **multimodal_kwargs["mm_processor_kwargs"])
         multimodal_group.add_argument(
-            "--disable-mm-preprocessor-cache",
-            **multimodal_kwargs["disable_mm_preprocessor_cache"])
+            "--mm-processor-cache-gb",
+            **multimodal_kwargs["mm_processor_cache_gb"])
+        multimodal_group.add_argument("--disable-mm-preprocessor-cache",
+                                      type=bool,
```

Comment:
This way it will default to `None` and you can check `if self.disable_mm_preprocessor_cache is not None` below, allowing you to warn if the user manually sets it to `False` too

```suggestion
                                      type=argparse.BooleanOptionalAction,
```